crates/sequencer/src/sequencer.rs

```diff
@@ -1016,22 +1016,29 @@ where
         &self,
         pg_connector: PostgresConnector,
     ) -> Result<(), anyhow::Error> {
+        let db_commitment = match pg_connector.get_last_commitment().await? {
+            Some(comm) => comm,
+            None => return Ok(()),
```

Comment:
Re-thinking my comment, i believe it's okay to return Ok here just to indicate they we're not making any adjustments here. Maybe place a comment around that return saying that we do nothing if postgres has no information about the last commitment.

Reply:
I know and I talked that with Esad as well and he suggested for us to sync-back the postgres in a different issue. Although I have no opposition to either ideas, my question would be, even in that is syncing necessary? Do we ever need all commitments and not just the last one from postgres outside of tests? Because if we only technically need the last one, the sequencer will eventually will publish a commitment and it will be the latest.